src/app/app.component.html

```diff
@@ -0,0 +1,53 @@
+<!-- {{ results | async | json }} -->
+
+<<!doctype html>
+<html>
+  <head>
+    <title>DeskDine - Add Menu Items</title>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap@4.3.1/dist/css/bootstrap.min.css"
+      integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"
+      crossorigin="anonymous" />
+  </head>
+
+  <body>
```

Comment:
you don't need to add all this <html> <head> <body> tag you can start using <div>